### ISSUE TITLE
selectedAssets cannot be preselected

### ISSUE THREAD

**Issue by iSevenDays** [DragonCherry/AssetsPickerViewController#75]
I found this config doesn't work as it is supposed

```
let config = AssetsPickerConfig()
config.selectedAssets = self.selectedAssets
```
Steps to reproduce
1. Open AssetsPickerViewController worskspace
2. Launch example Set Selected Assets Before Present
3. Click "Pick" and select 2 photos
4. Click "Pick" again

Expected result:
5. Selected photos from step 3 have checkmarks

Actual Result:
5. The photos are not pre-selected and I can select them indefinitely

Thanks

**Comment by Ava8** [comment]
I am having the same issue.

**Comment by DragonCherry** [comment]
Fixed on v2.7.8
Sorry for late response. Thank you!
